lib/tasks/base_path.rake

```diff
@@ -0,0 +1,24 @@
+namespace :base_path do
+  desc "Edits the base_path of a document, given its content_id and the new base_path"
+  task :edit, [:content_id, :base_path] => [:environment] do |_t, args|
+    content_id = args[:content_id]
+    base_path = args[:base_path]
+
+    begin
+      document = Document.find(content_id)
+    rescue DocumentFinder::RecordNotFound => e
+      puts "Error finding the document: #{e.inspect}"
+    end
+
+    if document
```

Comment:
If the exception were allowed to propagate, above, then this check wouldn't be needed.